### PR TITLE
Add some migration documentation for angularjs

### PR DESCRIPTION
**Pull request by internettrans**
See https://single-spa.slack.com/archives/CR2V2LYF6/p1614690233002100

### CODE REVIEW

File: website/versioned_docs/version-5.x/ecosystem-angularjs.md [hunk 1]
```diff
@@ -212,6 +212,79 @@ Migrating an existing AngularJS application to single-spa can be a tricky. Here
...
+- <script src="/main-angular-app.js"></script>
+```
+
+5. Modify the `<script>` in your main HTML file to load the angularjs app as a systemjs module instead of global variable:
+
+```diff
+- window.singleSpa.registerApplication({
+-   name: "legacyAngularjsApp",
+-   app: window.legacyAngularjsApp,
+-   app: function() { return System.import('@org/legacyAngularjsApp'); },
```
Comment (filoxo) [inline website/versioned_docs/version-5.x/ecosystem-angularjs.md]: This diff doesn't seem correct when rendered.

Screen Shot 2021-03-02 at 12 05 53 PM

Is this supposed to remove the registerApplication? If so, then there are duplicate `app: ` lines.

File: website/versioned_docs/version-5.x/ecosystem-angularjs.md [hunk 2]
```diff
@@ -212,6 +212,79 @@ Migrating an existing AngularJS application to single-spa can be a tricky. Here
...
+  {
+    "imports": {
+      "single-spa": "https://cdn.jsdelivr.net/npm/single-spa@5.9.1/lib/system/single-spa.min.js",
+      "@org/legacyAngularjsApp": "/main-angular-app.js",
+      "@org/new-microfrontend": "http://localhost:8080/new-microfrontend.js"
+    }
+  }
+</script>
+```
+3. Start a new microfrontend on the port in the import map. Go to the route for the new microfrontend and verify it is loaded.
```
Comment (filoxo) [inline website/versioned_docs/version-5.x/ecosystem-angularjs.md:288]: There's room here for maybe adding a link or callout to using create-single-spa. Up to you.

Comment (internettrans) [reply website/versioned_docs/version-5.x/ecosystem-angularjs.md:288]: I like it - will add.

File: website/versioned_docs/version-5.x/ecosystem-angularjs.md (hunk 1 above)
Comment (internettrans) [reply website/versioned_docs/version-5.x/ecosystem-angularjs.md]: Fixed

File: website/versioned_docs/version-5.x/ecosystem-angularjs.md (hunk 2 above)
Comment (internettrans) [reply website/versioned_docs/version-5.x/ecosystem-angularjs.md:288]: Updated

**Review by filoxo — APPROVED** [review]
🙈

**Pull request merged**